.github/workflows/jit.yml

```
@@ -96,7 +96,7 @@ jobs:
         if: runner.os == 'macOS'
         run: |
           brew install llvm@${{ matrix.llvm }}
-          export SDKROOT="$(xcrun --show-sdk-path)"
+          SDKROOT="$(xcrun --show-sdk-path)" \
           ./configure --enable-experimental-jit ${{ matrix.debug && '--with-pydebug' || '--enable-optimizations --with-lto' }}
```

Add an indent to make it clearer the line is being continued?
```suggestion
            ./configure --enable-experimental-jit ${{ matrix.debug && '--with-pydebug' || '--enable-optimizations --with-lto' }}
```